scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc

```diff
@@ -0,0 +1,57 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-provisioning-low-latency-workloads"]
+= Provisioning real-time and low latency workloads
+include::_attributes/common-attributes.adoc[]
+:context: cnf-provisioning-low-latency
+
+toc::[]
+
+Many organizations need high performance computing and low, predictable latency, especially in the financial and telecommunications industries.
+
+{product-title} provides the Node Tuning Operator to implement automatic tuning to achieve low latency performance and consistent response time for {product-title} applications.
+You use the performance profile configuration to make these changes.
+You can update the kernel to kernel-rt (real-time), reserve CPUs for cluster and operating system housekeeping duties, including pod infra containers, isolate CPUs for application containers to run the workloads, and disable unused CPUs to reduce power consumption.
+
+[NOTE]
+====
+When writing your applications, follow the general recommendations described in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_for_real_time/8/html-single/tuning_guide/index#chap-Application_Tuning_and_Deployment[Application tuning and deployment].
+====
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../scalability_and_performance/low_latency_tuning/cnf-tuning-low-latency-nodes-with-perf-profile.adoc#cnf-tuning-low-latency-nodes-with-perf-profile[Tuning nodes for low latency with the performance profile]
+
+include::modules/cnf-scheduling-workload-onto-worker-with-real-time-capabilities.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../nodes/scheduling/nodes-scheduler-node-selectors.adoc#nodes-pods-node-selectors[Placing pods on specific nodes using node selectors]
+
+* link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node[Assigning pods to nodes]
```

Comment:
```suggestion
* link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node[Assigning pods to nodes (Kubernetes documentation)]
```

Meets [this guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute) and let's a customer know that they'll be visiting an external site in advance.